sqlmesh/core/config/model.py

```diff
@@ -53,6 +53,7 @@ class ModelDefaultsConfig(BaseConfig):
     audits: t.Optional[t.List[FunctionCall]] = None
     optimize_query: t.Optional[bool] = None
     validate_query: t.Optional[bool] = None
+    lint_query: t.Optional[bool] = None
```

Comment:
If know that we plan on introducing more options in future, can we create `LintConfig` with just one property `enabled` so that we have easier time extending it in future without breaking end users?

Also why is this in `ModelConfig`? Doesn't this apply to audits and whatever other SQL we may introduce later?

Reply:
I also wonder whether `validate_query` is just another linting rule. So we'll have 2 rules already:
1. validate_query (needs a better name now)
2. no select star